docs/PR_GUIDE.md

```diff
@@ -0,0 +1,56 @@
+# Pull Request Guide
+
+## General development rules:
+
+- All the changes of the code base are implemented through pull requests (PR) on master or other branches
+- When all next release features are implemented in master and master is stable enough release branch is made from it
```

Comment:
When all the features for an upcoming release are implemented and are stable enough, a new release branch is made from master. After that, all app versions are updated in the master branch.

Reply:
(maybe link to the files with version here)